src/app/settings/views/Configuration/Security/SessionTimeout/SessionTimeout.test.tsx

```diff
@@ -0,0 +1,208 @@
+import { screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import configureStore from "redux-mock-store";
+
+import SessionTimeout, {
+  Labels as SessionTimeoutLabels,
+} from "./SessionTimeout";
+
+import { actions as configActions } from "app/store/config";
+import { ConfigNames } from "app/store/config/types";
+import type { RootState } from "app/store/root/types";
+import {
+  config as configFactory,
+  configState as configStateFactory,
+  rootState as rootStateFactory,
+} from "testing/factories";
+import { renderWithBrowserRouter } from "testing/utils";
+
+const mockStore = configureStore<RootState>();
+
+describe("SessionTimeout", () => {
+  let state: RootState;
+
+  beforeEach(() => {
+    state = rootStateFactory({
+      config: configStateFactory({
+        loaded: true,
+        loading: false,
+        items: [
+          configFactory({
+            name: ConfigNames.SESSION_LENGTH,
+            value: 1209600,
+          }),
+        ],
+      }),
+    });
```

Comment:
This is a good candidate for being added to default mock state in [getTestState](https://github.com/ndv99/maas-ui/blob/f92467af7f2fafa39d2a3a406c079056f3c8eaa1/src/testing/utils.tsx#L220).

Reply:
I'll add that in as part of this PR